util/downloadUrlByOS.ts

```diff
@@ -0,0 +1,28 @@
+import type { UserOS } from '../types/userOS';
+
+export const downloadUrlByOS = ({
+  userAgent,
+  userOS,
+  version,
+  bitness,
+}: {
+  userAgent?: string | undefined;
+  userOS: UserOS;
+  version: string;
+  bitness?: string;
+}): string => {
```

Comment:
Do you generally make a separate interface when taking an object as input? Or inline like this?

Reply:
A `type` actually, but yeah I didn't notice this, sorry 🙈